images/multus-cni/config/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install"
+  default     = []
```

Comment:
Theres no package defined? and the package is still in PR?
 - https://github.com/wolfi-dev/os/pull/15476

is that why the CI is passing?

Reply:
The PR is ready to review